extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.js

```diff
@@ -482,13 +489,42 @@ export class AmpLightboxViewer extends AMP.BaseElement {
   }
 
   /**
-   * Set up container listener.
+   * Set up event listeners.
    * @private
    */
-  setupContainerListener_() {
+  setupEventListeners_() {
     dev().assert(this.container_);
     const toggleControls = this.toggleControls_.bind(this);
-    listen(dev().assertElement(this.container_), 'click', toggleControls);
+    this.unlistenClick_ = listen(dev().assertElement(this.container_),
+        'click', toggleControls);
+
+    // iOS non-safari browsers do not reliably fire onResize on orientation
+    // change, so listen to orientationchange to trigger resize
+    const platform = Services.platformFor(this.win);
+    if (platform.isIos() && !platform.isSafari()) {
+      // The debounce is to force a 500ms delay due to Webkit bug #170595
```

Comment:
Please use an actual link instead of the bug number.

Reply:
This is done.